<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DockBandplan</class>
 <widget class="QDockWidget" name="DockBandplan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>464</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>327</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>524287</width>
    <height>524287</height>
   </size>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resources/icons.qrc">
    <normaloff>:/icons/icons/signal.svg</normaloff>:/icons/icons/signal.svg</iconset>
  </property>
  <property name="toolTip">
   <string>FFT performance and display settings</string>
  </property>
  <property name="allowedAreas">
   <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
  </property>
  <property name="windowTitle">
   <string>Band Plan Settings</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>360</width>
         <height>427</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>5</number>
          </property>
          <item row="6" column="2" colspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QPushButton" name="pbUseNone">
              <property name="text">
               <string>None</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pbUseAll">
              <property name="text">
               <string>All</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="4" column="0" colspan="2">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Modulation:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="2" colspan="2">
           <widget class="QListWidget" name="bandPlanUsesList">
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::MultiSelection</enum>
            </property>
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="2" colspan="2">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Use:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QListWidget" name="bandPlanRegionsList">
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::MultiSelection</enum>
            </property>
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Region:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QPushButton" name="pbRegionNone">
              <property name="text">
               <string>None</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pbRegionAll">
              <property name="text">
               <string>All</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="0" colspan="4">
           <widget class="QCheckBox" name="bandPlanCheckboxEnable">
            <property name="toolTip">
             <string>Enable Band Plan on bottom of spectrum</string>
            </property>
            <property name="text">
             <string>Enable Band Plan</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2" colspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QPushButton" name="pbCountryNone">
              <property name="text">
               <string>None</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pbCountryAll">
              <property name="text">
               <string>All</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="5" column="0" colspan="2">
           <widget class="QListWidget" name="bandPlanModulationsList">
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::MultiSelection</enum>
            </property>
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="7" column="0" colspan="4">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>253</width>
              <height>68</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="6" column="0" colspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QPushButton" name="pbModNone">
              <property name="text">
               <string>None</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pbModAll">
              <property name="text">
               <string>All</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="2" colspan="2">
           <widget class="QListWidget" name="bandPlanCountriesList">
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::MultiSelection</enum>
            </property>
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="2" colspan="2">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Country:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../../resources/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>bandPlanCheckboxEnable</sender>
   <signal>toggled(bool)</signal>
   <receiver>bandPlanCountriesList</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>429</y>
    </hint>
    <hint type="destinationlabel">
     <x>271</x>
     <y>485</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bandPlanCheckboxEnable</sender>
   <signal>toggled(bool)</signal>
   <receiver>bandPlanUsesList</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>238</x>
     <y>437</y>
    </hint>
    <hint type="destinationlabel">
     <x>258</x>
     <y>648</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbModAll</sender>
   <signal>clicked()</signal>
   <receiver>bandPlanModulationsList</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>212</x>
     <y>737</y>
    </hint>
    <hint type="destinationlabel">
     <x>111</x>
     <y>662</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbCountryNone</sender>
   <signal>clicked()</signal>
   <receiver>bandPlanCountriesList</receiver>
   <slot>clearSelection()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>578</y>
    </hint>
    <hint type="destinationlabel">
     <x>385</x>
     <y>537</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbRegionAll</sender>
   <signal>clicked()</signal>
   <receiver>bandPlanRegionsList</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>144</x>
     <y>578</y>
    </hint>
    <hint type="destinationlabel">
     <x>101</x>
     <y>504</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbRegionNone</sender>
   <signal>clicked()</signal>
   <receiver>bandPlanRegionsList</receiver>
   <slot>clearSelection()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>578</y>
    </hint>
    <hint type="destinationlabel">
     <x>101</x>
     <y>504</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbUseAll</sender>
   <signal>clicked()</signal>
   <receiver>bandPlanUsesList</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>388</x>
     <y>737</y>
    </hint>
    <hint type="destinationlabel">
     <x>385</x>
     <y>662</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bandPlanCheckboxEnable</sender>
   <signal>toggled(bool)</signal>
   <receiver>pbRegionAll</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>431</y>
    </hint>
    <hint type="destinationlabel">
     <x>165</x>
     <y>578</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bandPlanCheckboxEnable</sender>
   <signal>toggled(bool)</signal>
   <receiver>pbUseNone</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>435</y>
    </hint>
    <hint type="destinationlabel">
     <x>289</x>
     <y>753</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbModNone</sender>
   <signal>clicked()</signal>
   <receiver>bandPlanModulationsList</receiver>
   <slot>clearSelection()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>737</y>
    </hint>
    <hint type="destinationlabel">
     <x>111</x>
     <y>662</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bandPlanCheckboxEnable</sender>
   <signal>toggled(bool)</signal>
   <receiver>bandPlanRegionsList</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>433</y>
    </hint>
    <hint type="destinationlabel">
     <x>68</x>
     <y>485</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bandPlanCheckboxEnable</sender>
   <signal>toggled(bool)</signal>
   <receiver>bandPlanModulationsList</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>436</y>
    </hint>
    <hint type="destinationlabel">
     <x>46</x>
     <y>658</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbUseNone</sender>
   <signal>clicked()</signal>
   <receiver>bandPlanUsesList</receiver>
   <slot>clearSelection()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>300</x>
     <y>737</y>
    </hint>
    <hint type="destinationlabel">
     <x>385</x>
     <y>662</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbCountryAll</sender>
   <signal>clicked()</signal>
   <receiver>bandPlanCountriesList</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>385</x>
     <y>578</y>
    </hint>
    <hint type="destinationlabel">
     <x>385</x>
     <y>537</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bandPlanCheckboxEnable</sender>
   <signal>toggled(bool)</signal>
   <receiver>pbCountryNone</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>435</y>
    </hint>
    <hint type="destinationlabel">
     <x>256</x>
     <y>578</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bandPlanCheckboxEnable</sender>
   <signal>toggled(bool)</signal>
   <receiver>pbCountryAll</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>434</y>
    </hint>
    <hint type="destinationlabel">
     <x>356</x>
     <y>578</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bandPlanCheckboxEnable</sender>
   <signal>toggled(bool)</signal>
   <receiver>pbModAll</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>435</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>748</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bandPlanCheckboxEnable</sender>
   <signal>toggled(bool)</signal>
   <receiver>pbUseAll</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>332</x>
     <y>441</y>
    </hint>
    <hint type="destinationlabel">
     <x>361</x>
     <y>746</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bandPlanCheckboxEnable</sender>
   <signal>toggled(bool)</signal>
   <receiver>pbModNone</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>91</x>
     <y>438</y>
    </hint>
    <hint type="destinationlabel">
     <x>85</x>
     <y>738</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bandPlanCheckboxEnable</sender>
   <signal>toggled(bool)</signal>
   <receiver>pbRegionNone</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>36</x>
     <y>431</y>
    </hint>
    <hint type="destinationlabel">
     <x>27</x>
     <y>578</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
